examples/edcon/unicon.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE token  [
+        <!ENTITY unicon.en SYSTEM "unicon.en.shtml">
+        <!ENTITY style SYSTEM "shared.css">
+        ]>
+<ts:token xmlns:ts="http://tokenscript.org/2019/10/tokenscript"
+          xmlns="http://www.w3.org/1999/xhtml"
+          xmlns:xml="http://www.w3.org/XML/1998/namespace"
+          xsi:schemaLocation="http://tokenscript.org/2019/10/tokenscript http://tokenscript.org/2019/10/tokenscript.xsd"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          custodian="false"
+>
+    <ts:name>
+        <ts:plurals xml:lang="en">
+            <ts:string quantity="one">Ticket</ts:string>
+            <ts:string quantity="other">Tickets</ts:string>
+        </ts:plurals>
+    </ts:name>
+
+    <ts:contract name="unicon" interface="erc721">
+        <ts:address network="100">0xF6b8DD8Ba9996bEaE6Ad0eE3481F1E9cF080A9eB</ts:address>
+    </ts:contract>
+
+    <ts:origins>
+        <!-- Define the contract which holds the token that the user will use -->
+        <ts:ethereum contract="unicon">
+        </ts:ethereum>
+    </ts:origins>
+
+    <ts:cards>
+        <ts:token-card>
+            <style type="text/css">&style;</style>
+            <ts:view-iconified xml:lang="en">
+                &unicon.en;
+            </ts:view-iconified>
+            <ts:view xml:lang="en">
+                &unicon.en;
+            </ts:view>
+        </ts:token-card>
+    </ts:cards>
+
+    <ts:attribute-types>
+
+        <ts:attribute-type id="time" syntax="1.3.6.1.4.1.1466.115.121.1.24">
+            <ts:name>
+                <ts:string xml:lang="en">Time</ts:string>
+            </ts:name>
+            <ts:origins>
+                <ts:token-id bitmask="FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF00000000000000000000000000" as="utf8">
+                </ts:token-id>
+            </ts:origins>
+        </ts:attribute-type>
+
+        <!-- TODO get location & venue, if only one then remove this attribute and hardcode it into the UI -->
+        <ts:attribute-type id="locality" oid="2.5.4.7" syntax="1.3.6.1.4.1.1466.115.121.1.15">
+            <ts:name>
+                <ts:string xml:lang="en">City</ts:string>
+            </ts:name>
+            <ts:origins>
+                <ts:token-id bitmask="000000000000000000000000000000000000FF00000000000000000000000000" as="uint">
+                    <ts:mapping>
+                        <ts:option key="1">
+                            <ts:value xml:lang="en">Vienna</ts:value>
+                        </ts:option>
+                    </ts:mapping>
+                </ts:token-id>
+            </ts:origins>
+        </ts:attribute-type>
+
+        <ts:attribute-type id="venue" syntax="1.3.6.1.4.1.1466.115.121.1.15">
+            <ts:name>
```

Comment:
@zhangzhongnan928 if you know the location, let me know

Reply:
TBC, 
They haven't paid to the venue.

Reply:
it is
Vienna University of Economics and Business
Welthandelspl. 1, 1020 Wien, Austria